internal/eval/commands.go

```diff
@@ -1206,6 +1206,45 @@ var (
 		Arity:    3,
 		KeySpecs: KeySpecs{BeginIndex: 1},
 	}
+	linsertCmdMeta = DiceCmdMeta{
+		Name: "LINSERT",
+		Info: `
+		Usage:
+			LINSERT key <BEFORE | AFTER> pivot element
+		Info:
+			Inserts element in the list stored at key either before or after the reference value pivot.
+			When key does not exist, it is considered an empty list and no operation is performed.
+			An error is returned when key exists but does not hold a list value.
+		Returns:
+			Integer - the list length after a successful insert operation.
+			0 when the key doesn't exist.
+			-1 when the pivot wasn't found.
+		`,
+		Eval:     evalLINSERT,
+		Arity:    4,
```

Comment:
as per redis command info it is 5, I guess they count the command name as well. 
```suggestion
		Arity:    5,
```

Reply:
Done.